crates/sqlexec/src/remote/client.rs

```diff
@@ -193,14 +180,33 @@ impl RemoteClient {
         }
 
         let mut dst: Endpoint = dst.try_into()?;
-        if let Some(tls_conf) = tls_conf {
-            metadata.insert(CA_CERT_PATH, tls_conf.ca_cert_path.parse()?);
-            metadata.insert(DOMAIN, tls_conf.domain.parse()?);
-            let ca = std::fs::read_to_string(tls_conf.ca_cert_path)?;
+
+        let mut api_url = "https://console.glaredb.com/api/internal/authenticate/client";
```

Comment:
I need more context here - the reason we can't pass it down from args or otherwise, easily, is because it would require an end user to set it.

The logic I added (which I admit looks a bit strange) is to construct the API URL from the connection string they use.,

so for example a user will do:

```
\open glaredb://user:pass@org.remote.glaredb.com:6443/deployment
```

and from that we know to use the `console.glaredb.com/api`.

Reply:
I think it'd be better to have it be settable via commandline args and then overridable with defaults if people don't set it, rather than hardcode our cloud deployment at this point that feels deep in the middle of the code.

Reply:
I used an arg to accomplish this (cloud_addr)